types/node-fetch/index.d.ts

```diff
@@ -165,12 +166,12 @@ export type ResponseType =
     | "opaqueredirect";
 
 export interface ResponseInit {
-    headers?: HeaderInit;
+    headers?: HeadersInit;
     status: number;
     statusText?: string;
 }
 
-export type HeaderInit = Headers | string[];
+export type HeadersInit = Headers | {[key: string]: string};
```

Comment:
```suggestion
export type HeadersInit = Headers | string[][] | { [key: string]: string };
```

Reply:
Ah yeah, I missed that code block in node-fetch initially which is why I removed it. Added it back  as per your suggestions, thanks for the catch! Also, I made this a breaking change, but not sure if it is such since it is a fix?

Reply:
@JasonLi914 The version number in these types should reflect which version of node-fetch they represent. The DefinitelyTyped release team handle any other versioning.